frontend/cypress.config.js

```diff
@@ -8,29 +8,14 @@ module.exports = defineConfig({
   watchForFileChanges: false,
   e2e: {
     setupNodeEvents(on, config) {
-      // implement node event listeners here
-      config.specPattern = [
-        "cypress/e2e/login.cy.js",
-        "cypress/e2e/home.cy.js",
-        "cypress/e2e/patientEntry.cy.js",
-        "cypress/e2e/orderEntity.cy.js",
-        "cypress/e2e/workplan.cy.js",
-        "cypress/e2e/nonConform.cy.js",
-        "cypress/e2e/result.cy.js",
-        "cypress/e2e/validation.cy.js",
-        "cypress/e2e/modifyOrder.cy.js",
-        "cypress/e2e/report.cy.js",
-        "cypress/e2e/batchOrderEntry.cy.js",
-        "cypress/e2e/dashboard.cy.js",
-        "cypress/e2e/labNumberManagement.cy.js",
-        "cypress/e2e/AdminE2E/MenuConfig/globalMenuConfig.cy.js",
-      ];
+      // Listeners
       return config;
     },
     baseUrl: "https://localhost",
     testIsolation: false,
     env: {
       STARTUP_WAIT_MILLISECONDS: 300000,
     },
+    specPattern: "cypress/e2e/**/*.cy.js", // Automatically includes all test specs
```

Comment:
This will definately fail as For our use case, the tests are not run in isolattion of each other, so Cypress has to depend on a pre-defined order of execution